cmd/hivechain/genesis.go

```diff
@@ -100,6 +102,16 @@ func (cfg *generatorConfig) createChainConfig() *params.ChainConfig {
 			chaincfg.CancunTime = &timestamp
 		case "prague":
 			chaincfg.PragueTime = &timestamp
+			if cfg.blobSchedule != nil {
```

Comment:
Can you remove this for now? We will add it back when go-ethereum has the change to add BlobSchedule on master.

Reply:
I removed the change!